components/do_verify_email/do_verify_email.jsx

```diff
@@ -6,15 +6,21 @@ import React from 'react';
 import {FormattedMessage} from 'react-intl';
 import {browserHistory} from 'react-router';
 
-import {verifyEmail} from 'actions/user_actions.jsx';
-
 import logoImage from 'images/logo.png';
 
 import BackButton from 'components/common/back_button.jsx';
 
-import LoadingScreen from './loading_screen.jsx';
+import LoadingScreen from 'components/loading_screen.jsx';
+
+export default class DoVerifyEmail extends React.PureComponent {
+    static propTypes = {
+
+        /**
+         * Object with validation parameters given in link
+         */
+        location: PropTypes.object.isRequired
+    }
```

Comment:
I guess you forgot below. That might also be one of the reason why you're test (patterned to `EditOAuthApp`) is not working properly.
```
        actions: PropTypes.shape({
            verifyUserEmail: PropTypes.func.isRequired
        }).isRequired
```